src/components/Carts/Cart/cart.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import useCartHooks from "../../../redux/Shopping/shopping-actions";
+import { Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  Button,
+  Row
+} from "reactstrap";
+
+const CartItem = ({ item }) => {
+  const [input, setInput] = useState(item.qty);
+  const {adjustItemQty , removeFromCart}  = useCartHooks();
+
+  const onChangeHandler = (e) => {
+    setInput(e.target.value);
+    adjustItemQty(item.id, e.target.value);
+  };
```

Comment:
Destructuring elements

```suggestion
    const { value } = e.target
    setInput(value);
    adjustItemQty(item.id, value);
  };
```

Reply:
Done